src/Shared/runtime/Http2/Hpack/H2StaticTable.cs

```diff
@@ -1,6 +1,5 @@
 // Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0.
```

Comment:
FYI @Tratcher this will trigger our sync bot right? How does it handle license headers today? The eventual change (moving to MIT headers) should simplify things, right?

Reply:
Yeah, the bot will pick up any src difference in these shared/runtime files. Ideally runtime would have it's licence changed first. Otherwise we'll want to disable the bot until both sides are updated.

Reply:
Or exclude the src/shared/runtime directory and deal with it later?

Reply:
Runtime should *already* be MIT?

Reply:
Nope. https://github.com/dotnet/runtime/blob/4f84429fb44801b841c052505935a3cd14da18e2/src/libraries/Common/src/System/Net/Http/aspnetcore/Http2/Hpack/H2StaticTable.cs#L1-L3

Reply:
@sharwell do you want to update the licenses in the runtime repo to MIT or should I? See the readme for syncing the changes.
https://github.com/dotnet/runtime/blob/master/src/libraries/Common/src/System/Net/Http/aspnetcore/ReadMe.SharedCode.md

Reply:
Why don't we just sync the runtime after our transition is complete. It's still manual (even if tracked automatically).

Reply:
I want to limit how long they're out of sync. It makes syncing other changes harder.